api/v1/checluster_types.go

```diff
@@ -475,9 +475,6 @@ type CheClusterSpecAuth struct {
 	// See link:https://github.com/brancz/kube-rbac-proxy[kube-rbac-proxy] or link:https://github.com/openshift/kube-rbac-proxy[openshift/kube-rbac-proxy]
 	// +optional
 	GatewayAuthorizationSidecarImage string `json:"gatewayAuthorizationSidecarImage,omitempty"`
-	// Header Rewrite Proxy sidecar image is used to properly set authorization header.
-	// See link:https://github.com/che-incubator/header-rewrite-proxy[header-rewrite-proxy]
-	GatewayHeaderRewriteSidecarImage string `json:"gatewayHeaderRewriteSidecarImage,omitempty"`
```

Comment:
We can't remove/rename any fields. It will break backward compatibility. 
Just mark it as deprecated.

Reply:
done